windows/src/test/unit-tests/kmcomp/test.bat

```diff
@@ -52,25 +52,33 @@ call :should-pass "#2241: &CasedKeys (chars)" test_casedkeys_chars.kmn || goto :
 
 call :should-fail "#2241: &CasedKeys (mnemonic 1)" test_casedkeys_mnemonic_1.kmn || goto :eof
 call :should-fail "#2241: &CasedKeys (mnemonic 2)" test_casedkeys_mnemonic_2.kmn || goto :eof
-call :should-pass "#2241: &CasedKeys (mnemonic 3)" test_casedkeys_mnemonic_3.kmn || goto :eof
+call :should-fail "#2241: &CasedKeys (mnemonic 3)" test_casedkeys_mnemonic_3.kmn || goto :eof
```

Comment:
Is this test supposed to have been always failing?

Reply:
Yes, I believe so. I think that the should-pass test was actually FAILing but if you look at (line 77)[https://github.com/keymanapp/keyman/pull/5963/files/83eb8395d69683ac33ce878067556d8e9706bbd1#diff-27713d4507c7fb0f20b201e10ea21858b08f16e2abee737ae3490f18d405bad6R77] you can see it was masking the failure. Ooops.